node/src/Pos/Wallet/Web/ClientTypes/Instances.hs

```diff
@@ -120,6 +108,19 @@ instance Buildable CAccountInit where
         caInitWId
         caInitMeta
 
+instance Buildable NewPaymentBatchInit where
+    build NewPaymentBatchInit{..} =
+        bprint ("{ from="%build
+                %" to="%(later mapBuilder)
+                %" policy="%build
+                %" }")
+        npbFrom
+        npbTo
+        npbPolicy
+
+instance Buildable (SecureLog NewPaymentBatchInit) where
```

Comment:
hope this is secure?

Reply:
I don't think we are leaking any sensible information here, so my understanding is that it's secure, but probably @Martoon-00 or @flyingleafe can confirm this.